storybook/stories/playground/index.js

```diff
@@ -21,8 +21,6 @@ import { registerCoreBlocks } from '@wordpress/block-library';
 import '@wordpress/format-library';
 
 /* eslint-disable no-restricted-syntax */
-import '@wordpress/components/build-style/style.css';
-import '@wordpress/block-editor/build-style/style.css';
 import '@wordpress/block-library/build-style/style.css';
```

Comment:
If this workflow is desired, we could apply the same technique to the editor/library styles as well.

Reply:
I think it's perfectly fine to use sources everywhere. We do it for stories where we import JS source files directly, so we can follow the same approach for styles.